Source/WebCore/rendering/GridTrackSizingAlgorithm.cpp

```diff
@@ -1103,6 +1115,10 @@ LayoutUnit GridTrackSizingAlgorithmStrategy::minContentContributionForGridItem(R
 
     if (updateOverridingContainingBlockContentSizeForGridItem(gridItem, gridItemInlineDirection)) {
```

Comment:
Right after we call `updateOverridingContainingBlockContentSizeForGridItem` we trigger the invalidation logic numerous times.

I think we can just move the invalidation logic inside of `updateOverridingContainingBlockContentSizeForGridItem`.

Reply:
I'm not really sure if we should be adding code to that method that does things other than updating the containing block size for the item